### PR TITLE
Update proxmoxve.markdown deprecated instructions updated

### PR DESCRIPTION
**Pull request by ted1735**
deprecated instructions updated

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/proxmoxve.markdown [hunk 1]
```diff
@@ -22,6 +22,8 @@ After configuring this integration, the binary sensors automatically appear.
 
 <div class='note'>
 You should have at least one VM or container entry configured, else this integration won't do anything.
+  These instructions are deprecated and better suited to follow this link to get up and running.
```
Comment (frenck) [inline source/_integrations/proxmoxve.markdown:25]: We should not send user to third-party resources to set up an core integration.

If our existing instructions needs to be update, please update this page instead.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/proxmoxve.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/proxmoxve.markdown:25]: Just one note: I am not aware this integration has been deprecated. Could you point me to a resource that made this decision?

**Comment by ted1735** [comment]
I can definitely agree not to go to 3rd. I'm very new to suggesting changes
or markdown on git. I should rephrase the integration isn't deprecated.
Just the instructions. The resource I pointed to worked flawlessly after me
trying to edit yaml for two hours! Once I stumbled upon the resource it was
completely automated!

**Comment by frenck** [comment]
I've looked at the instructions you've linked and those apply to a custom integration, not Home Assistant itself; thus therefore do not apply here.

The Home Assistant project does not support or recommend the use of custom integrations.

Going to close this PR for the above reasoning.

../Frenck

**Pull request closed without merging**